apps/gp2-server/src/controllers/reminder.controller.ts

```diff
@@ -18,7 +18,16 @@ export default class ReminderController {
         href: gp2Routing.outputs({}).output({
           outputId: reminder.data.outputId,
         }).$,
-        description: `**${reminder.data.statusChangedBy}** in ${reminder.data.associationType} **${reminder.data.associationName}** published a **${reminder.data.documentType}** output: "${reminder.data.title}".`,
+        description:
+          reminder.entity === 'Output'
+            ? `**${reminder.data.statusChangedBy}** in ${reminder.data.associationType} **${reminder.data.associationName}** published a **${reminder.data.documentType}** output: "${reminder.data.title}".`
+            : `**${reminder.data.statusChangedBy}** on **${
+                reminder.data.associationName
+              }** published a new ${
+                reminder.data.associationType
+              } ${reminder.data.documentType.toLowerCase()} output version: "${
```

Comment:
I believe this document type is in bold as well

<img width="739" alt="Screenshot 2024-02-08 at 07 34 42" src="https://github.com/yldio/asap-hub/assets/16595804/ecaccf49-6ef2-4f3e-be34-7a12446cf1a1">